gnomad/resources/grch38/reference_data.py

```diff
@@ -361,6 +391,28 @@ def _import_ensembl_interval(path) -> hl.Table:
     },
 )
 
+gtex_rsem = VersionedTableResource(
+    default_version="v10",
+    versions={
+        "v7": GnomadPublicTableResource(
+            path="gs://gnomad-public-requester-pays/resources/grch38/gtex_rsem/gtex_rsem_v7.mt",
+            import_func=_import_gtex_rsem,
+            import_args={  # TODO: update path
+                "gtex_path": "gs://gnomad-qin/bulk-gex_v7_rna-seq_GTEx_Analysis_2016-01-15_v7_RSEMv1.2.22_transcript_tpm.txt.gz",
+                "meta_path": "gs://gnomad-qin/annotations_v7_GTEx_v7_Annotations_SampleAttributesDS.txt.gz",
+            },
+        ),
+        "v10": GnomadPublicTableResource(
```

Comment:
v10 is my presumed name, we can also delete it for now and make the default_version to v7.

Reply:
I think for now we should only add the resource for v7. We can add the new resource when we get it